.github/TOOL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,31 @@
+_The following form is designed to provide information for your tool that should be included in the effort to verify the Rust standard library. Please note that the tool will need to be supported if it is to be included._
+
+## Tool Name
+_Please enter your tool name here._
+
+## Description
+_Please enter a description for your tool and any information you deem relevant._
+
+### Tool Information
+
+* [ ] Does the tool perform Rust verification? 
+* [ ] Does the tool deal with *unsafe* Rust code? 
+* [ ] Does the tool run independently in CI? 
+* [ ] Is the tool open source?
+* [ ] Is the tool under development? 
+* [ ] Will you or your team be able to provide support for the tool? 
+
+### Licenses
+<!-- Please list the licenses that the tool uses and can be used under. -->
+
+### Steps to Use the Tool
+
+1. [First Step]
+2. [Second Step]
+3. [and so on...]
+
+### Artifacts
```

Comment:
non-blocking: I'd separate this section into two 
1. ### tutorial
 _Include an example of using the tool to perform verification _

2. ### artifacts
_Include links, papers, blog posts about your tool_

Reply:
my bad. I actually missed a hashtag. fixing.

Reply:
fixed.